src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -114,7 +115,7 @@ const styles = StyleSheet.create({
     paddingTop: 7,
     paddingBottom: 7,
     borderBottomWidth: 1,
-    borderBottomColor: COLORS.alto
+    borderBottomColor: colors.alto
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant has been updated. Make sure this new color constant is defined and correctly represents the intended color.
```diff
-    borderBottomColor: COLORS.alto
+    borderBottomColor: colors.alto
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->